realm/realm-library/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -779,7 +779,7 @@ public static void populateForDistinctFieldsOrder(Realm realm, long numberOfBloc
     }
 
     public static void awaitOrFail(CountDownLatch latch) {
-        awaitOrFail(latch, 60);
+        awaitOrFail(latch, 300);
```

Comment:
Doggone it!  Yes.  That's a mistake.  I sometimes need that for my local tests, but it shouldn't be standard.  WIll fix.